tests/test_index.py

```diff
@@ -132,11 +132,9 @@ def test_linear_index_gather():
     assert matches[0][1] == ss2
 
     matches = lidx.gather(ss47)
-    assert len(matches) == 2
+    assert len(matches) == 1
     assert matches[0][0] == 1.0
     assert matches[0][1] == ss47
-    assert round(matches[1][0], 2) == 0.49
-    assert matches[1][1] == ss63
```

Comment:
This test is a good example of the 'weird' behavior: if doing gather with `ss47`, it should match `ss47` completely, and nothing will be left to match other sigs. But the test is asserting that there is a second match (`ss63`), at 0.49 containment. That's... not gather, that's a regular search using containment.